tests/test_myco_external_matcher.py

```diff
@@ -52,28 +52,30 @@ def test_publish_simulation_id(self):
         self.matcher.myco_ext_conn.publish_json.assert_called_once_with(
             channel, {"simulation_id": self.matcher.simulation_id})
 
-    def test_publish_event_tick_myco(self):
+    def test_event_tick(self):
         data = {"event": "tick"}
-        self.matcher.publish_event_tick_myco()
+        self.matcher.event_tick()
         self.matcher.myco_ext_conn.publish_json.assert_called_once_with(
             self.events_channel, data)
+        self.matcher.event_tick(is_it_time_for_external_tick=False)
+        assert self.matcher.myco_ext_conn.publish_json.call_count == 1
```

Comment:
Wait, if is it not time for an external tick, `self.matcher.myco_ext_conn.publish_json.call_count` should be 0, right?

Reply:
We are calling the event tick above and testing wether it was called (it should)
So at this point, when we call it again it won't invoke the redis publisher but the counter is still 1 (not 2)

Reply:
I see. Please add a comment that explains that. Thanks!